tests/components/account.spec.mjs

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import Account from '../../src/components/account';
+
+describe('components/account.mjs', () => {
+  describe('class Account', () => {
+    describe('constructor()', () => {
+      it('should initialize', () => {
+        const { id }: Account = new Account();
```

Comment:
new Account() duplicate (x8), beforeEach to init

Reply:
Good observation, didn't see it